onnxruntime/test/providers/qnn/conv_test.cc

```diff
@@ -397,218 +410,328 @@ TEST_F(QnnCPUBackendTests, ConvTranspose1Df32_DynamicWeights_DefaultBias) {
 
 // Check that QNN compiles DQ -> Conv -> Q as a single unit.
 // Tests bias as a dynamic input.
-TEST_F(QnnHTPBackendTests, ConvU8S32_bias_dynamic_input) {
-  RunHTPConvOpTest<uint8_t>("Conv",
-                            TestInputDef<float>({1, 1, 5, 5}, false, 0.0f, 10.0f),   // Random dynamic input
-                            TestInputDef<float>({1, 1, 3, 3}, true, -10.0f, 10.0f),  // Random static input
-                            TestInputDef<float>({1}, false, {2.0f}),                 // Dynamic bias = 2.0f
-                            {1, 1},                                                  // Strides
-                            {0, 0, 0, 0},                                            // Pads
-                            {1, 1},                                                  // Dilations
-                            "NOTSET",
-                            ExpectedEPNodeAssignment::All);
+TEST_F(QnnHTPBackendTests, ConvU8U8S32_bias_dynamic_input) {
+  RunHTPConvOpTest<uint8_t, uint8_t>("Conv",
+                                     TestInputDef<float>({1, 1, 5, 5}, false, 0.0f, 10.0f),   // Random dynamic input
+                                     TestInputDef<float>({1, 1, 3, 3}, true, -10.0f, 10.0f),  // Random static input
+                                     TestInputDef<float>({1}, false, {2.0f}),                 // Dynamic bias
+                                     {1, 1},                                                  // Strides
+                                     {0, 0, 0, 0},                                            // Pads
+                                     {1, 1},                                                  // Dilations
+                                     "NOTSET",
+                                     ExpectedEPNodeAssignment::All);
+}
+
+// Tests 16-bit QDQ Conv with dynamic weights and bias (uses QNN's Conv2d)
+// TODO(adrianlizarraga): Inaccuracy detected for output 'output', element 0.
+// Output quant params: scale=6585.83056640625, zero_point=65535.
+// Expected val: -67.35833740234375
+// QNN QDQ val: -431602400 (err 431602336)
+// CPU QDQ val: 0 (err 67.35833740234375)
```

Comment:
Naively both QNN and CPU results seem way off an expected value of -67, but I'm not sure how to connect this info to the actual test. a bit more info in the comment might be required. 

will the issues causing the tests to be disabled be resolved prior to checkin?

Reply:
This was the result of using bad test inputs that produced a very large floating-point range of outputs that saturated the quantized range. Specifically, the output values ranged from -431602208 to 202.58. This resulted in an output scale of 6585.83, which severely limited the granularity of values that could be produced by the QDQ models.

I updated the tests to use more reasonable test inputs.

Reply:
do the comments on lines 428 - 430 still apply to the updated input?

Reply:
The comments on lines 428 were updated. I'm not sure if you're seeing an older commit? This is the current comment:
```c++
// Tests 16-bit QDQ Conv with dynamic weights and bias (uses QNN's Conv2d)
// TODO: Inaccuracy detected for output 'output', element 0.
// Output quant params: scale=0.0040235077030956745, zero_point=0.
// Expected val: 87.354057312011719
// QNN QDQ val: 0 (err 87.354057312011719)
// CPU QDQ val: 87.3583984375 (err 0.00434112548828125)
```